subsys/bootloader/include/dummy_values/pm_config.h

```diff
@@ -0,0 +1,7 @@
+/* DUMMY FILE ONLY TO BE USED FOR TESTING */
+#ifndef PM_CONFIG_H__
+#define PM_CONFIG_H__
+#define PM_B0_ADDRESS 0x0
```

Comment:
Could these hard-coded addresses be copied from Kconfig/DTS?

Reply:
They are dummy values, so their correctness is not important.

There is no way to extract this information when building this sample stand-alone as they only generated by partition manager. But partition manager is not executed when only a single image is included in the build.

One alternative would be to expose kconfig options for these values when b0 is built stand alon. This way the user can avoid partition manager all together and still have an easy way of configuring the bootloader to work with their project configuration.